hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -426,7 +427,7 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 				case metav1.ConditionFalse:
 					condition.Status = metav1.ConditionTrue
 					condition.Reason = failingCond.Reason
-					condition.Message = ""
+					condition.Message = "Cluster version is not failing"
```

Comment:
Something like the `ClusterVersion` resource status message is likely more appropriate here:
```
Cluster version is 4.11.0-rc.4
```

Reply:
That is the message on the `Progressing` condition of the ClusterVersion resource.  CPO will have to propagate that.  I'll make the change.

Reply:
I was just thinking that you could create the message since I think you do have access to the version in status.